app/controllers/api/v2/report_templates_controller.rb

```diff
@@ -115,15 +116,23 @@ def export
       def generate
```

Comment:
https://edgeapi.rubyonrails.org/classes/ActionController/Live.html suggests you need to implement `stream` to actually send the data (in a separate thread).